cloudant-sync-datastore-core/src/test/java/com/cloudant/sync/internal/replication/ReplicationTestBase.java

```diff
@@ -238,14 +240,30 @@ protected void assertCookieInterceptorPresent(ReplicatorBuilder p, String expect
         Assert.assertEquals(CookieInterceptor.class, reqIList.get(1).getClass());
         Assert.assertEquals(1, respIList.size());
         Assert.assertEquals(CookieInterceptor.class, respIList.get(0).getClass());
-        CookieInterceptor ci = (CookieInterceptor)reqIList.get(1);
-        Field srbField = CookieInterceptor.class.getDeclaredField("sessionRequestBody");
+        CookieInterceptorBase ci = (CookieInterceptorBase)reqIList.get(1);
```

Comment:
I think this was correct before. `CookieInterceptor` is the `_session` interceptor, `CookieInterceptorBase` would assert for either `CookieInterceptor` or `IamCookieInterceptor`.

Reply:
This failed when I changed it back, maybe because `getDeclaredField` only works for the class the field is declared in and not the subclasses?

Reply:
Sorry, I missed the assert for `CookieInterceptor` on the earlier line. Yes, this is fine.